ngx_http_secure_token_m3u8.c

```diff
@@ -1,9 +1,14 @@
 #include "ngx_http_secure_token_encrypt_uri.h"
 #include "ngx_http_secure_token_m3u8.h"
 
-static u_char encryption_key_tag[] = "EXT-X-KEY";
-static u_char ext_media_tag[] = "EXT-X-MEDIA";
-static u_char ext_i_frame_tag[] = "EXT-X-I-FRAME-STREAM-INF";
+static ngx_str_t uri_tags[] = {
```

Comment:
there are some unrelated commits below this point, better open a new branch/pull so that only your changes will be there.